package.json

```diff
@@ -18,5 +18,8 @@
     "stylelint": "^14.5.3",
     "stylelint-config-standard-scss": "^3.0.0",
     "stylelint-order": "^5.0.0"
+  },
+  "dependencies": {
+    "@nasa-jpl/stellar": "^1.0.5"
```

Comment:
The package should depend on itself

Reply:
Removed in [b39dd47](https://github.com/nasa-jpl/stellar/pull/6/commits/b39dd4759e133330fd1bba3ba4cdafa9faa30aa7).